RELEASE.md

```diff
@@ -244,6 +236,34 @@ Ping team, ping community, ping friends - the latest Deephaven has been released
 The release branches serve a purpose for kicking off CI jobs, but aren't special in other ways.
 Sometime after a release, old release branches can be safely deleted.
 
+### 15. Version bump in preparation of the next release.
+
+Say we just did release 0.31. The next expected release is 0.32.  We update the repository with a bump to all files that
+mention the version explicitly.  The current list of files that need to be updated is listed below:
+
+```
+#
+# Edit files, updating from `X.Y` to `X.(Y+1)`
+#
+authorization-codegen/protoc-gen-contextual-auth-wiring
+authorization-codegen/protoc-gen-service-auth-wiring
+buildSrc/src/main/groovy/io.deephaven.common-conventions.gradle
+py/client-ticking/README.md
+py/client-ticking/setup.py
+py/client/README.md
+py/client/pydeephaven/__init__.py
+py/client/setup.py
+py/embedded-server/deephaven_server/__init__.py
+py/server/deephaven/__init__.py
+R/rdeephaven/DESCRIPTION
+```
+
+This leaves the files "ready" for the next regular release, and also ensures any build done from
+a developer for testing of latest is not confused with the code just released.
+
+In the case of a patch release these would need to be updated to a different version, eg, to 0.31.1 in the case of a patch
```

Comment:
Maybe replace "eg, to 0.31.1 in the case of a patch
release after 0.31, instead of how they were left at 0.32 in this step." with "eg, to 0.31.1 from 0.31."